### DISCUSSION TITLE
Is there any generally accepted method for polling when an update becomes available?

### DISCUSSION THREAD

**Discussion by famgala** [jolpica/jolpica-f1#98] [Q&A] [answered]
I am designing an app which will pull data down for reach race. I only need to pull the data once per race, but I am not sure the best way to handle this since there is no way to know when the data is available.

**Comment by jolpica** [comment]
We don't have anyway to check when the data was last updated currently.

If you need to pull down the latest race data then polling the endpoint `https://api.jolpi.ca/ergast/f1/current/last/results/` once an hour on Sunday / Monday should be fine as that endpoint is heavily utilised so will be cached regularly.

**Reply by martinleppen** [reply]
Not really related to the API development itself, but I was wondering when the data for the Brazilian GP will be available?

**Reply by jolpica** [reply, marked as answer]
Sorry, there was an error in the data upload this week and I didn't spot it.

Should be available for you now.
